libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateEnemyProjectile.cpp

```diff
@@ -79,9 +79,12 @@ namespace ecs
             default: createBasicProjectile(entity); break;
         };
         entity.addComponent<Position>(pos.x, pos.y + 20)
-            .addComponent<Collidable>()
-            .addComponent<EnemyProjectile>(enemy.get()->getComponent<Networkable>().id);
-
+            .addComponent<Collidable>();
+        if (enemy->contains<Networkable>()) {
+            entity.addComponent<EnemyProjectile>(enemy->getComponent<Networkable>().id);
+        } else {
+            entity.addComponent<EnemyProjectile>(enemy->getComponent<Enemy>().enemyType);
```

Comment:
Same as Allied

Reply:
To add the AlliedProjectileComponent with id of the player in solo mod